Caboodle/DeviceInfo/DeviceInfo.netstandard.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace Microsoft.Caboodle
+{
+    public static partial class DeviceInfo
```

Comment:
These should be properties, not methods.

Reply:
private implementations.

Reply:
AHHHHHHHHHHHHHHHHHHHHHHHHHHHHH